src/main/java/org/gridsuite/filter/server/dto/expertfilter/expertrule/AbstractExpertRule.java

```diff
@@ -56,7 +61,7 @@ public abstract class AbstractExpertRule {
     @Schema(description = "Rules")
     private List<AbstractExpertRule> rules;
 
-    public abstract boolean evaluateRule(Identifiable<?> identifiable);
+    public abstract boolean evaluateRule(Identifiable<?> identifiable, FilterService filterService, Map<UUID, FilterEquipments> mapFilters);
```

Comment:
Maybe `mapFilters` should be named something like `cachedUUIDFilters`? Because we add it to every rule and use it only for FilterUuid rule

Reply:
Renamed